app/views/shared/email/_myCollection_block.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if Rails.env.production? %>
+    <p>Simply <%= link_to 'sign up', "https://www.artsy.net/signup?submissionId=#{@submission.id}", target: :_blank %> or <%= link_to 'log in', "https://www.artsy.net/login?submissionId=#{@submission.id}", target: :_blank %> to Artsy with the same email you used for this submission and visit our app.</p>
+<% else %>
+    <p>Simply <%= link_to 'sign up', "https://staging.artsy.net/signup?submissionId=#{@submission.id}", target: :_blank %> or <%= link_to 'log in', "https://staging.artsy.net/login?submissionId=#{@submission.id}", target: :_blank %> to Artsy with the same email you used for this submission and visit our app.</p>
```

Comment:
The indentation here should be 2, right?

Reply:
right! I'm not sure it affects the rendered html though. I have tried different indentations, but the email template seems to be unaffected by that. 

I'll change it to 2 regardless 👍